website/docs/advanced-concepts/access-control/roles.md

```diff
@@ -0,0 +1,247 @@
+
+#### Edit
+
+**Edit** allows the user to modify the existing applications, pages, database/API queries and JS Objects in a workspace. You can edit the app name, pages, and their order, color, icon, theme etc. Edit also enables access to deploy an app and connect to a Git repository. 
+Edit permission can be given at different hierarchical levels -
+
+##### Workspace 
+
+When you grant the **Create** permission to a workspace, it provides access to edit resources only in that particular workspace. This also automatically enables permission to view, edit the existing pages, database/API queries and JS Objects.
+
+
+##### Application
+
+If you want the user to have edit access only to a specific application in a workspace, open the respective workspace dropdown and check the edit permission adjacent to the respective application. **Edit** for just an application allows you to edit resources like page, actions (APIs, queries, JSObjects), Git connection etc. 
+
+##### Page
+
+At a more granular level, you can restrict **Edit** permission to a specific page and its resources within an application by checking the box next to the page name. You can edit the page name, clone the page, set it as the home page, and toggle their visibility.
+
+#### View 
+
+**View** only grants a user the access to view and interact with the deployed applications in a workspace. You can't make any changes as you won't be able to access the canvas if you only have the View permission. It applies to all the hierarchical levels in a workspace.
+
+#### Delete
+
+**Delete** gives the user an access to delete the existing applications, pages, or actions (APIs, queries, JSObjects) in a workspace. With the delete permission, one can't edit or create resources, but, the user has the access to delete it.
+
+Delete permission behaves differently at various hierarchical levels -
+
+##### Workspace 
+
+ If you give delete permission to the workspace, It gives access to remove any application from the respective workspace. 
+
+##### Application
+
+If you want the user to have delete access only to a specific application in a workspace, open the respective workspace's dropdown and check the delete permission adjacent to the respective application. **Delete** for just an application allows you to delete resources like page, actions (APIs, queries, JSObjects).
+
+##### Page
+
+**Delete** access for a particular page allows you to delete the respective page or the resources in the page like APIs, queries, JSObjects. 
+
+#### Export
+
+**Export** allows a user to export the applications in a workspace. If a user has export permission enabled, he can also view the applications in the workspace.
+This permission can also be enabled for a particular application, allowing the user to export and view the respective application only.
+
+#### Make public
+
+**Make Public** gives the user an ability to make the application public or private in a workspace.
```

Comment:
Same like export, make public gives view access as well

Reply:
As discussed, we'll just mention what this permission does and not the cascading effect it has.